nodejs/eks/cluster.ts

```diff
@@ -314,22 +314,40 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
 
     const roleMappings = pulumi.all([pulumi.output(args.roleMappings || []), instanceRoleMappings])
         .apply(([mappings, instanceMappings]) => {
-            return jsyaml.safeDump([...mappings, ...instanceMappings].map(m => ({
+            mappings.map(m => {
+                if (!checkArnExists(pulumi.output(m.roleArn))) {
+                    throw new Error(`The IAM role ARN ${m.roleArn} could not be properly retrieved or does not exist`);
+                }
+            });
```

Comment:
I don't think this part of the change is necessary - from the original bug report that part of the issue will be fixed more properly by https://github.com/pulumi/pulumi-terraform/issues/262.  We don't in general want to `.get` every resource id the user mentions to us just to verify they didn't make a mistake.

Reply:
Ok